rotkehlchen/utils/misc.py

```diff
@@ -394,3 +396,9 @@ def create_order_by_rules_list(
     order_by_attribute = data['order_by_attributes'] if data['order_by_attributes'] is not None else [default_order_by_field]  # noqa: E501
     ascending = data['ascending'] if data['ascending'] is not None else [is_ascending_by_default]
     return list(zip(order_by_attribute, ascending))
+
+
+class NftLpHandling(SerializableEnumMixin):
+    ALL_NFTS = auto()
```

Comment:
Since this is not used, perhaps remove it.

Reply:
As I said in the other thing this is used instead of having Optional[NftLpHandling], which I think is a bit cleaner. So is fine to keep I guess